packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -132,7 +132,10 @@ depth = model.infer_image(raw_img) # HxW raw depth map in numpy
 const diffusers_default = (model: ModelData) => [
 	`from diffusers import DiffusionPipeline
 
-pipeline = DiffusionPipeline.from_pretrained("${model.id}")`,
+pipeline = DiffusionPipeline.from_pretrained("${model.id}")
+
+prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
```

Comment:
Maybe a simpler prompt will work better as it will be applied on all type diffusion models? (cc @apolinario)

```suggestion
prompt = "Astronaut in a jungle"
```

Reply:
Adding those few keywords just exceptionally boost the quality.

Reply:
Ok then!